README.md

```diff
@@ -1,16 +1,33 @@
-# Errors and Debugging Roundtable Activity
+# Intro to Debugging Roundtable Activity
```

Comment:
The name of the roundtable was updated to Intro to Debugging (from Errors and Debugging back in C18)